### PR TITLE
changed specs from openai to groq ai

### PR DESCRIPTION
**Pull request by enochval**
@KseniiaRiabova @Valeriusdev please assist to review this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,10 +57,10 @@ implement to enhance this app, if time permits.
 - [ ] This is a purely frontend application. No backend is required.
 - [ ] You may use any languages, tools, or libraries you prefer when designing and building this app.
 - [ ] You may **_NOT_** use AI-base solution generators like GitHub CoPilot.
-- [ ] The app should facilitate connectivity to the OpenAI API for color recommendations.
+- [ ] The app should facilitate connectivity to the Groq OpenAI API for color recommendations.
```
Comment (franeCode) [inline README.md:60]: You probably forgot to delete OpenAI from here.

Comment (enochval) [reply README.md:60]: I thought Groq has compatibility(integration) with OpenAI. see the api url https://api.groq.com/openai/v1/chat/completions. what do you think @franeCode ?

Comment (franeCode) [reply README.md:60]: Yes, you are right. Just some thoughts about name consistency.

Comment (enochval) [reply README.md:60]: I agree, I will remove it for consistency sake.

**Pull request merged**